setup.py

```diff
@@ -36,9 +36,9 @@
     install_requires=[
         "numpy",
         "scipy",
-        "torch",
+        "torch>=2.0.1",
```

Comment:
This seems like a pretty hard requirement for a lot of folks? Maybe we can keep this requirement, but make a comment to revert to an older version of this repo?